README.md

```diff
@@ -1,71 +1,138 @@
-the UNIX design philosophy of doing something specific and doing it well. 
-Nomad integrates with Terraform, Consul, and Vault for provisioning, service 
-discovery, and secrets management.
-
-For more information, see the [introduction section](https://www.nomadproject.io/intro)
-of the Nomad website.
-
-Getting Started & Documentation
+Overview
+-------------------------------
+
+Nomad is a flexible, lightweight, easy-to-use, performant workload orchestrator that deploys:
+
+* [Containers](https://www.nomadproject.io/docs/drivers/docker.html)
+* [Legacy applications](https://www.nomadproject.io/docs/drivers/exec.html)
+* [Virtual machines](https://www.nomadproject.io/docs/drivers/qemu.html)
+
+Nomad enables developers to use declarative infrastructure-as-code for deploying their applications (jobs).  Nomad uses bin packing to efficiently schedule jobs and optimize for resource utilization.  Nomad is supported on macOS, Windows, and Linux.
+
+Nomad is widely adopted and used in production by PagerDuty, Target, Citadel, Trivago, SAP, Pandora, Roblox, eBay, Deluxe Entertainment, and more.   
+
+* **Deploy Containers and Legacy Applications**: Nomad’s flexibility as an orchestrator enables an organization to run containers, legacy, and batch applications together on the same infrastructure.  Nomad brings core orchestration benefits to legacy applications without needing to containerize via pluggable task drivers.
+
+* **Simple & Reliable**:  Nomad runs as a single 75MB binary and is entirely self contained - combining resource management and scheduling into a single system.  Nomad does not require any external services for storage or coordination.  Nomad automatically handles application, node, and driver failures.  Nomad is distributed and resilient, using leader election and state replication to provide high availability in the event of failures.   
+
+* **Device Plugins & GPU Support**: Nomad offers built-in support for GPU workloads such as machine learning (ML) and artificial intelligence (AI).  Nomad uses device plugins to automatically detect and utilize resources from hardware devices such as GPU, FPGAs, and TPUs.
+
+* **Federation for Multi-Region, Multi-Cloud**: Nomad was designed to support infrastructure at a global scale.  Nomad supports federation out-of-the-box and can deploy jobs across multiple regions and clouds.
+
+* **Proven Scalability**: Nomad is optimistically concurrent, which increases throughput and reduces latency for workloads.  Nomad has been proven to scale to clusters of 10K+ nodes in real-world production environments.
+
+* **HashiCorp Ecosystem**: Nomad integrates seamlessly with Terraform, Consul, Vault for provisioning, service discovery, and secrets management.
+
+Getting Started
 -------------------------------
 
-All documentation is available on the [Nomad website](https://www.nomadproject.io).
+Get started with Nomad quickly in a sandbox environment on the public cloud or on your computer.
+
+* Local
```

Comment:
Could we add another getting started option without the need for vagrant? We can list prerequisites like having Docker installed. It should be as simple as 1) Download the binary 2) Run `nomad agent -dev` 3) Run `nomad init` 4 ) Run `nomad run example.nomad`

Reply:
Do you know if there's documentation?  Don't recall seeing those instructions anywhere.  A little hesitant to prescribe anything in the README that isn't reflected officially on www.nomadproject.io as well...

Reply:
This would need us to publish a new section to https://www.nomadproject.io/intro/getting-started/install.html. Fine to punt on this for now and address in a future PR